server/api.py

```diff
@@ -1088,6 +1092,31 @@ def get_tournament_matches_by_slug(
     return 200, Match.objects.filter(tournament=tournament).order_by("time")
 
 
+@api.get(
```

Comment:
Would it be better to include this information in the `UserProfile`? The list of team IDs for which a player is an admin?

Reply:
Yep so thought about it first.. So then ill have to send a map of sorts for that case. Cause pras might be cap of l2f in mgr and I might be cap of l2f for sgr

so then i need to send a map of tournament mapping to the teamIDs a user is eligible to submit